CHANGELOG.md

```diff
@@ -83,6 +83,20 @@
 
     The method `NewRelic::Agent::External.start_segment` has been deprecated as of Ruby Agent 6.0.0. This method is now removed.
 
+  * **Added testing and support for the following gem versions**
+
+    - activemerchant 1.121.0
+    - bunny 2.19.0
+    - excon 0.85.0
+    - mongo 2.14.0, 2.15.1
+    - padrino 0.15.1
+    - resque 2.1.0
+    - sequel 5.48.0
+    - yajl-ruby 1.4.1
+
+  * **This version adds support for ARM64/Graviton2 platform using Ruby 3.0.2+**
```

Comment:
We did run a lot of tests on 2.7.4 if you want to add that too! Totally up to you though :)

Reply:
Did Erik have any opinions about where in the list of changes he'd like the ARM64 support called out?

Reply:
I just used the wording exactly as erik suggested, which included the latest ruby version since that is what our support requirement is. And he didn't mention where in the list it should be, I'm not sure that matters? but i can ask!

Reply:
Ah cool. Since this was from him and he didn't specify it's probably not important!